packages/postgres-database/src/simcore_postgres_database/utils_services_environments.py

```diff
@@ -0,0 +1,39 @@
+from typing import TypeAlias
+
+from sqlalchemy.sql import select
+
+from ._protocols import DBConnection
+from .models.services_environments import VENDOR_SECRET_PREFIX, services_vendor_secrets
+
+# This constraint is to avoid deserialization issues after substitution!
+VendorSecret: TypeAlias = bool | int | float | str
+
+
+async def get_vendor_secrets(
+    conn: DBConnection,
+    vendor_service_key: str,  # NOTE: ServiceKey is defined in model_library
```

Comment:
Does it complain if you try to do something like this?
```suggestion
    vendor_service_key: "ServiceKey",  # NOTE: ServiceKey is defined in model_library
```
```suggestion
    vendor_service_key: str,  # NOTE: ServiceKey is defined in model_library
```